index.json

```diff
@@ -1,7 +1,11 @@
 {
-    "version": 14,
+    "version": 15,
```

Comment:
Pretty sure version shouldn't be bumped as this is a new translation

cc: @Yimura